app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -102,7 +120,8 @@ class LeaseUpApplicationsPage extends React.Component {
 
   render () {
     const listing = this.props.listing
-
+    console.log(listing)
```

Comment:
probably just here because it's still WIP, but wanted to give you a heads up just in case

Reply:
removed!